bedrock/base/templates/macros.html

```diff
@@ -349,7 +349,9 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
               <input id="id-input" name="phone-or-email" type="text" required>
             </div>
             <div class="form-submit">
-              <button type="submit" class="button-flat" data-download-os="Android">{{ _('Send') }}</button>
+              <button type="submit" class="button-flat"
+              {% if platform == 'android' %} data-download-os="Android"
+              {% elif platform == 'ios' %} data-download-os="iOS" {% endif %}>{{ _('Send') }}</button>
```

Comment:
While we're not actively using it on any current pages, there is another possible option here: `platform == 'select'`. In this instance the user gets to select either Android or iOS as their chosen platform before submitting the form. Is this something you would like to add attributes for now proactively? Happy to defer to what you decide here.